server/opts.go

```diff
@@ -1518,6 +1520,10 @@ func parseJetStream(v interface{}, opts *Options, errors *[]error, warnings *[]e
 				opts.JetStreamMaxMemory = mv.(int64)
 			case "max_file_store", "max_file":
 				opts.JetStreamMaxStore = mv.(int64)
+			case "domain":
+				opts.JetStreamDomain = mv.(string)
+			case "enable", "enabled":
```

Comment:
Not clear on that: is that because you want to be able to specify a Domain, but not have JetStream enabled per-se? By the way, in earlier discussions with @matthiashanel, I think he thought of this setting be in the leafnodes{} accept block, no? (but may be better to lift it higher)

Reply:
Yes originally thought it would be around LNs but upon further thinking it belongs in the jetstream block. 
The enable is to support mixed mode clusters that all have active solicited LNs and want a separate JS domain for the LN cluster.